docs/questions/native-editor/referencing-saved-questions-in-queries.md

```diff
@@ -46,59 +46,55 @@ We'll select that question, and Metabase will update our code with the question'
 
 ```
 SELECT count(*)
-FROM {% raw %}{{#5}}{% endraw %}
+FROM {% raw %}{{#5-gizmo-orders-in-2019}}{% endraw %}
 ```
 
 This query returns the number of rows in our saved question.
 
-## Saved question as a Common Table Expression (CTE)
+## Model, table, or saved question as a Common Table Expression (CTE)
 
 The same syntax can be used in [Common Table Expressions (CTEs)](https://en.wikipedia.org/wiki/Hierarchical_and_recursive_queries_in_SQL#Common_table_expression) (with SQL databases that support CTEs):
```

Comment:
```suggestion
The same syntax can be used in [Common Table Expressions (CTEs)](https://www.metabase.com/learn/sql-questions/sql-cte) (with SQL databases that support CTEs):
```

Looks like this link was supposed to go to the Learn article (we can get rid of the footer link too).